libcamera_app.hpp

```diff
@@ -445,7 +451,7 @@ class LibcameraApp
 		StreamDimensions(video_stream_, w, h, stride);
 		return video_stream_;
 	}
-	void QueueRequest(BufferMap const &buffers)
+	void QueueRequest(CompletedRequest const &buffer_set)
 	{
```

Comment:
Sorry, @davidplowman, but would you consider renaming `buffer_set` to `completed_request` to be consistent with the new name?

With or without that change, I think this is good to merge.

Reply:
Sure, I did wonder a bit about that... let me fix it up and then we can hit the big green button.